posthog/test/base.py

```diff
@@ -142,10 +144,20 @@ def setUpTestData(cls):
             _setup_test_data(cls)
 
     def setUp(self):
+
+        if get_instance_setting("PERSON_ON_EVENTS_ENABLED"):
```

Comment:
@tiina303 not sure if this adversely affects anything for AsyncMigration tests, but open to suggestions.

The problem is: We want person on events tests to run when the instance setting is set. BUT, in all tests, the async migration isn't marked as completed, so person-on-events aren't enabled.

This is causing the smoke test to pass (when it should fail)